internal/service/logs.go

```diff
@@ -47,8 +50,14 @@ func (l *Logs) GetLogs(ctx *context.Context, request *logs.GetLogsRequest) (int6
 			continue
 		}
 
+		verboseEnabled := false
+		if (*ctx).Value(constant.VerboseEnabledKey) != nil {
+			verboseEnabled = (*ctx).Value(constant.VerboseEnabledKey).(bool)
```

Comment:
If verbose is enabled, simply remove `DEBUG` from `restrictedLogLevels` to avoid complicated conditions